frontend/packages/dev-console/src/components/pipelines/PipelineAugmentRuns.tsx

```diff
@@ -48,7 +42,7 @@ const PipelineAugmentRuns: React.FC<PipelineAugmentRunsProps> = ({
   const children = inject(props.children, {
     ...props,
     resources: { pipeline: { data: resourceData } },
-    filters: { ...props.filters, ...paramFilters },
+    filters: { ...props.filters },
```

Comment:
Do you need this? it's in props which is spread above.
Try removing it.

Reply:
It works even after removing the filters.